.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -21,6 +21,13 @@ body:
       label: Summary
       description: Describe the issue in 1 or 2 sentences
       placeholder: Clearly describe what the expected behavior is vs. what is actually happening. This should be as short as possible, while still communicating all the necessary information. If your summary is just 'X library/method isn't working', then you need to continue debugging yourself and provide more information.
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Managed or bare?
```

Comment:
In a lot of cases where workflow is the issue users do no know that they are using bare, maybe it would be better if we detected that value and included it in the doctor command output?

Reply:
I think we should actually do both.